### ISSUE TITLE
AGP 3.5.0 - Could not find the AndroidManifest.xml file, using  generation folder

### ISSUE THREAD

**Issue by YarikSOffice** [androidannotations/androidannotations#2229]
Getting the same issue with #2128 using the latest Android Gradle Plugin 3.5.0. 

**Android Gradle Plugin version**: 3.5.0-beta03
**AndroidAnnotations version**: 4.6.0
**Android compile SDK version**: 28
**build.gradle**:
```
    annotationProcessor "org.androidannotations:androidannotations:4.6.0"
    implementation "org.androidannotations:androidannotations-api:4.6.0"
```
**Stacktrace in case of AA crash**:
```
error: Could not find the AndroidManifest.xml file, using  generation folder [**/build/generated/ap_generated_sources/debug/out])
```

**Temporary Workaround**
Add the following code in build.gradle file for each module that uses AA library.
```
    defaultConfig {
        javaCompileOptions {
            annotationProcessorOptions {
                arguments = [
                        "androidManifestFile": "$projectDir/src/main/AndroidManifest.xml".toString()
                ]
            }
        }
    }
```

**Comment by dodgex** [comment]
I opened a PR to support current beta and alpha versions of AGP.

**Comment by prathamApp** [comment]
Same here. Not working for 3.5.0 but fortunately for 3.4.2

**Comment by WonderCsabo** [comment]
The fix is now available in the 4.7.0-SNAPSHOT build.
